packages/carbon-web-components/src/components/pagination/pagination.ts

```diff
@@ -25,6 +25,18 @@ import { carbonElement as customElement } from '../../globals/decorators/carbon-
  * Pagination UI.
  *
  * @element cds-pagination
+ * @csspart left-container - The left container. Usage: `cds-pagination::part(left-container)`
+ * @csspart text - The generic shadow part for all text elements. Usage: `cds-pagination::part(text)`
+ * @csspart text-left - The left pagination text. Usage: `cds-pagination::part(text-left)`
```

Comment:
Sorry, I wasn't clear before. We've been using a convention inspired by [BEM](https://getbem.com/naming/#modifier), where the variant or modifier is named with a double-dash separator. For instance, see @marcelojcs 's recent change to the [code-snippet element](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/feat/v2-shadow-parts/packages/carbon-web-components/src/components/code-snippet/code-snippet.ts#L436). If you aren't familiar with BEM, that page is well worth the read. It'll also help make sense of many of the class names you see in the Carbon projects, as well as in AEM. 
```suggestion
 * @csspart text--left - The left pagination text. Usage: `cds-pagination::part(text--left)`
```